lib/src/builder.dart

```diff
@@ -135,6 +136,20 @@ class AggregateTestBuilder extends Builder {
     // Don't generate an empty aggregate test.
     if (imports.isEmpty) return;
 
+    if (_config.randomizeAggregation) {
+      log.fine('randomizing test order');
+      final randomizedMains = mains.toString().split(';\n')
+        ..removeWhere((element) => element.isEmpty);
+      var seed = _config.testShuffleSeed;
```

Comment:
If we change the field for this to be a String, then we'd move the logic that picks a random seed when the value is equal to `random` to right here instead of in the config class.